internal/pkg/service/buffer/store/key/key.go

```diff
@@ -20,10 +20,14 @@ type MappingKey struct {
 	RevisionID int `json:"revisionId" validate:"min=1,max=100"`
 }
 
-type SliceKey struct {
+type FileKey struct {
 	ExportKey
-	FileID  string
-	SliceID string
+	FileID string `json:"fileId" validate:"required"`
```

Comment:
Este musim doriesit nejake veci pre Honzu, tak to pozriem podrobnejsie,
ale zatial som si vsimol toto:

Mohlo by to byt namiesto `string` radsej `time.Time`? Vid `RecordKey`.

No najmä pouzi prosim `key.FormatTime` metodu, ktoru sme uz riesili.

Reply:
Ahá, ok.

Reply:
Akorát teda asi tady budu potřebovat poradit  😕  https://github.com/keboola/keboola-as-code/blob/f90c300998b6692e83277542216ba74279db2228/internal/pkg/service/buffer/store/schema/file.go#L47-L52
Mám to File ID před odesláním do tý `Key()` metody převést na string nebo je s tím třeba udělat něco jinýho?

![CleanShot 2022-12-05 at 15 25 53](https://user-images.githubusercontent.com/215660/205661806-9202fca9-66fa-41d7-b779-bfa9c0734770.jpg)

Reply:
A tady obdobně: https://github.com/keboola/keboola-as-code/blob/f90c300998b6692e83277542216ba74279db2228/internal/pkg/service/buffer/store/schema/record.go#L68-L69
![image](https://user-images.githubusercontent.com/215660/205663987-c4805e61-e0c3-4667-afb1-a6e62b979255.png)

Reply:
jj to je to miesto kde sa to prevadza na string key a teda sa ma pouzit ta funkcia `FormatTime`.

Reply:
(inak to `IsZero` super, nevedel som, ze to ma taku metodu)